src/components/Home.js

```diff
@@ -1,11 +1,64 @@
 import React from 'react';
 import { Link } from 'react-router';
 import CircularProgressbar from 'react-circular-progressbar';
+import userSettings from '../containers/userSettings';
+import helpers from './helpers'
 
-export default class Home extends React.Component {
+class Home extends React.Component {
   constructor() {
     super();
-  }
+    this.state = {
+      events: helpers,
+      pushEvents: null,
+    }
+}
+
+goToRepos() {
+  this.context.router.push('/repos');
+}
+
+grabUserInfo() {
+  fetch(`https://api.github.com/users/bcgodfrey91/events?page=0&callback`, {
+    method: 'GET'
+  })
+  .then((res) => {return res.json(); })
+  .then((response) => { this.setState({events: response}) })
+  .catch(() => { alert('Please try again.')})
+}
+
+getPushEvent() {
+  const { events } = this.state
+  const pushEv = events.filter((ghEvent) => ghEvent.type==='PushEvent')
+  this.setState({ pushEvents: pushEv})
+}
+
+getOpenedPullRequests() {
+  const { events } = this.state
+  const pullReq = events.filter((ghEvent) => ghEvent.type==='PullRequestEvent')
+  const openedPullRequests = pullReq.filter((obj) => obj.payload.action==='opened')
+  return openedPullRequests.length
+}
+
+getIssuesCreated() {
+  const { events } = this.state
+  const issues = events.filter((ghEvent) => ghEvent.type==='IssuesEvent')
+  const openedIssues = issues.filter((obj) => obj.payload.action==='opened')
+  return openedIssues.length
+}
+
+getIssuesClosed() {
+  const { events } = this.state
+  const issues = events.filter((ghEvent) => ghEvent.type==='IssuesEvent')
+  const closedIssues = issues.filter((obj) => obj.payload.action==='closed')
+  return closedIssues.length
```

Comment:
I think there's gotta be a way three 3 'get issues' functions can be ONE function that takes an issue type as an argument. Stand by for my attempt.